docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -35,12 +43,12 @@ specified as either the node id or node name.
 
 [source,js]
 --------------------------------------------------
-$ curl -XGET 'http://localhost:9200/_cluster/allocation/explain' -d '{
+{
```

Comment:
I don't know how to test this since I don't know the name or ID of a node currently in the cluster to which the shard is assigned